### DISCUSSION TITLE
Support different mouse pointers

### DISCUSSION THREAD

**Discussion by jaguililla** [kovidgoyal/kitty#6769] [Ideas]
It would be great to have extra SGR escape codes (like the ones for [colored and styled underlines](https://sw.kovidgoyal.net/kitty/underlines)) to allow the mouse pointer to change on different text segments.

I.e.: switching to the 'pointer' mouse cursor when mouse tracking is enabled, and the cursor is over a text that act as a button.

If there is a way to achieve that now, could you please give some hints on how to proceed?

Thanks!

**Comment by kovidgoyal** [comment]
dup of #6711
